(ATAC-seq chromatin accessibility analysis pipeline used to Chen et. al)

The pipeline aims to streamline processing ATAC-seq data from raw fastq files generated by Sequencer, read alignment, read counting and normalization, peak chracterization, and downstream differential analysis. It includes several developed packages to make publication-ready plots. Most of the analyses are built on top of existing tools and/or packages publicly available. Please find the reference link as indicated. Here, the pipeline demonstration consisits of two parts, seqeunce alignment and read count in Bash script (part I) and differential analysis and plotting in R script (Part II):

Part I. Alignment and read counting (Bash):
-----
- [bcl2fastq](https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html): Demultiplexing from raw sequencing reads in bcl files from the Sequencer.
- [fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)/[multiqc](https://multiqc.info/): Quality control for sequencing reads
- [Trimmomatic](http://www.usadellab.org/cms/?page=trimmomatic): Read trimming for paired-end fastq file
- [bowtie2](https://bowtie-bio.sourceforge.net/bowtie2/index.shtml): Read alignment of paired fastq files to acquire BAM file
- [samtools](https://github.com/samtools/samtools): Sorting and indexing of mapped BAM file
- [MACS3](https://github.com/macs3-project/MACS): Read alignment sensitive for spliced transcripts to acquire BAM file
- featureCounts](https://academic.oup.com/bioinformatics/article/30/7/923/232889): Counting reads aligned to each peak for downstream analysis
- [bamCoverage](https://github.com/deeptools/deepTools/blob/master/docs/content/tools/bamCoverage.rst): Normalization of mapped reads to generate bigwig file for visualization
- [HOMER](http://homer.salk.edu/homer/ngs/quantification.html): 

Part II. Differential analysis (R):
-----
- [DESeq2](https://github.com/mikelove/DESeq2): Differential analysis taking counts file as input for normalization and statistical inference
- [ggplot2](https://github.com/tidyverse/ggplot2): For basic QC plotting
- [dplyr](https://github.com/tidyverse/dplyr): Organize count data for downstream analysis
- [CHIPSeeker](https://guangchuangyu.github.io/software/ChIPseeker/): Gene set enrichment analysis (GSEA) to identify top-regulated pathways
- [EnhancedVolcano](https://github.com/kevinblighe/EnhancedVolcano): Figure plotting of log2-fold-change to P-value
